config/initializers/flickraw.rb

```diff
@@ -0,0 +1,2 @@
+FlickRaw.api_key=ENV['FLICKR_API_KEY']
+FlickRaw.shared_secret=ENV['FLICKR_SHARED_SECRET']
```

Comment:
Surrounding space missing for operator `=`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Final newline missing.